api_tests/src/actors/actor.ts

```diff
@@ -807,6 +801,7 @@ export class Actor {
 
         do {
             swapEntity = await this.swap.fetchDetails();
+            console.log(JSON.stringify(swapEntity, null, 2));
```

Comment:
This should not be part of this patch set :)

Reply:
Debugging code does not go into production, bad Tobin no biscuit.